package.json

```diff
@@ -33,7 +33,8 @@
     "jest": "cross-env NODE_ENV=test LOG_LEVEL=error MOCK_REDIS=1 FEED_URL_INTERVAL_MS=200 jest -c jest.config.js --",
     "jest:e2e": "jest -c jest.config.e2e.js --",
     "coverage": "cross-env NODE_ENV=test LOG_LEVEL=silent MOCK_REDIS=1 FEED_URL_INTERVAL_MS=200 jest -c jest.config.js --collectCoverage --",
-    "jest-watch": "cross-env MOCK_REDIS=1 jest --watch --",
+    "jest-watch": "cross-env MOCK_REDIS=1 jest -c jest.config.js --watch --",
+    "jest-update": "cross-env MOCK_REDIS=1 jest -c jest.config.js --u --",
```

Comment:
`--u` should either be `--updateSnapshot` or `-u`, see https://jestjs.io/docs/cli#--updatesnapshot.  NOTE: flags like this are always done with a single `-` and single letter, or double '--` and full words.

Let's use `--updateSnapshot` to help document this.